### DISCUSSION TITLE
How to get powerline effect in groupbox?

### DISCUSSION THREAD

**Discussion by vedantnn71** [qtile/qtile#2406] [Q&A] [answered]
# Issue description
I want to create a powerline effect in qtile like this  👇
![powerline-effect](https://user-images.githubusercontent.com/70624701/115029778-f465a900-9ee3-11eb-9369-01ea1ecf3686.png)
How do I do this in qtile? or Should I use polybar instead?
# Qtile version
0.17.0

**Comment by ramnes** [comment, marked as answer]
By using powerline? :)

https://powerline.readthedocs.io/en/master/usage/wm-widgets.html#qtile-widget

You can also just throw a `▶` or similar UTF8 character in a TextBox and play with it. This is what I'm doing for my diagonal separators here: https://github.com/ramnes/qtile-config/blob/0e23205/config.py#L213-L214

**Comment by elParaguayo** [comment]
OK - I'm very (very) late here but qtile-extras provides easy support for poweline:
https://qtile-extras.readthedocs.io/en/stable/manual/ref/decorations.html#powerlinedecoration
